### PR TITLE
Add CumulusCI

### LINKED ISSUE #138: Add CumulusCI
Add CumulusCI to this project in order to simplify Dev/QA org creation: https://cumulusci.readthedocs.io/en/latest/

### PR DESCRIPTION
**Pull request by Julian88Tex**
Closes #138

### CODE REVIEW

**Comment by VinceFINET** [comment]
hi @Julian88Tex 

I am working on using CumulusCI for OrgCheck on the branch called "cumulusci"
I started with an automatic task that tries to extract the metadata package and install it in a scratch org ad launch all the apex tests.

As you know OrgCheck does not include any apex class but that was a nice first use case to implement cumulusci and make it work.

**Comment by Julian88Tex** [comment]
@VinceFINET Nice! When you say "automatic task" do you mean that you used one of the OOTB tasks like `install_managed`? I'd be happy to run a test to validate :). I'm more of a QA than a dev :-P

File: robot/OrgCheck/tests/create_contact.robot [hunk 1]
```diff
@@ -0,0 +1,59 @@
+*** Settings ***
```
Comment (Julian88Tex) [inline robot/OrgCheck/tests/create_contact.robot:1]: @VinceFINET Feel free to leave this in if you like, but it's just an example Robot task more for demoing how Robot works within CCI.

**Comment by VinceFINET** [comment]
I created a branch for CumulusCI --> https://github.com/VinceFINET/OrgCheck/tree/cumulusci
@Julian88Tex does it correspond to your idea??

**Comment by Julian88Tex** [comment]
@VinceFINET yup! Seems like it is similar to this PR so I'll go ahead and close this one out. Looks like yours goes a step further and adds in a github action for apex tests which is great! Feel free to add me as a reviewer on a PR for it if you like.

**Pull request closed without merging**